intern/service/boss/baseversion.py

```diff
@@ -181,8 +185,12 @@ def build_cutout_url(
         if len(id_list) > 0:
             urlWithParams += '?filter=' + self.convert_int_list_to_comma_sep_str(id_list)
 
-        if no_cache:
-            urlWithParams += '?no-cache=true'
+        if access_mode == 'no_cache':
```

Comment:
if we trust the `access_mode` variable (passed by the caller) we could just do
`urlWithParams += '?access_mode=' + access_mode`

Reply:
I think since there are only those three possible options. I don't mind writing them out.